components/scream/scripts/jenkins/jenkins_common.sh

```diff
@@ -27,7 +27,9 @@ if [ $skip_testing -eq 0 ]; then
       SUBMIT="" # We don't submit AT runs
   fi
 
-  ./scream/components/scream/scripts/gather-all-data "./scripts/test-all-scream \$compiler -p -i -m \$machine $SUBMIT" -l -m $SCREAM_MACHINE
+  BASELINES_DIR=/home/projects/e3sm/scream/pr-autotester/master-baselines/$SCREAM_MACHINE
```

Comment:
This field should be implemented via the machine dictionary. This path is not going to work for the various machines being tested outside of sandia.

Reply:
Good catch. I have to think about how to handle this, cause I do need to pass something to `--baseline-dir`.